tags/developerguide.tag

```diff
@@ -2,5 +2,10 @@ type: text
 
 ---
 
-**You want to contribute to Geyser? This page has some useful things to start contributing to Geyser.**
-https://github.com/GeyserMC/Geyser/wiki/Developer-Guide
\ No newline at end of file
+You want to contribute to Geyser? This page has some useful things to start contributing to Geyser and links to some of the tools we use.
+https://github.com/GeyserMC/Geyser/wiki/Developer-Guide
+
+And here are some useful Java guides:
+ - https://docs.oracle.com/javase/tutorial/
+ - https://www.codecademy.com/learn/learn-java
+ - http://mooc.fi/courses/2013/programming-part-1/
```

Comment:
Maybe add a section for video guides because I found the Alex Lee ones quite helpful and some people prefer those

Reply:
These were pulled from a mee6 tag, we can remove them or add any links you suggest.